addons/sale_stock/models/sale_order.py

```diff
@@ -260,6 +260,14 @@ def _show_cancel_wizard(self):
                 return True
         return res
 
+    def _get_additional_template_data(self):
+        # OVERRIDE
+        self.ensure_one()
+        data = super(SaleOrder, self)._get_additional_template_data()
+        if self.incoterm:
+            data.append((_("Incoterm"), self.incoterm.code))
```

Comment:
Do not forget that new translated strings in stable versions means you have to re-export the POT file ;)